coops-ui/src/components/AnnualReport/Directors.vue

```diff
@@ -802,11 +802,6 @@ export default class Directors extends Mixins(DateMixin, ExternalMixin, EntityFi
               directors[i].officer.prevLastName = directors[i].officer.lastName
               directors[i].officer.prevMiddleInitial = directors[i].officer.middleInitial
 
-              // if this is a Coop, copy delivery address to mailing address - directors only have delivery addresses
```

Comment:
I assume the BE doesn't expect a director mailing address anywhere (for a Coop). Please do as much testing as possible (and/or leave notes in the ticket for QA to verify this). Are the output PDFs OK?

Reply:
Correct, it does not expect a mailing address. I can certainly try and verify the outputs. This code was removed because it was redundant. Originally a work around i imagine. Will verify all of the above.

Reply:
It appears the back end outputs a mailing address as well for Coops currently, though i'm not sure if the back end tickets for the outputs have been done for COD. I will inform QA. Changes to the BE outputs are outside the scope of this ticket i think.

Reply:
OK, thanks for that.